test/e2e/specs/published-content/likes__comment.ts

```diff
@@ -38,7 +38,7 @@ describe( 'Likes: Comment', function () {
 	let commentToBeUnliked: NewCommentResponse;
 	let restAPIClient: RestAPIClient;
 
-	beforeAll( async function () {
+	it( 'Setup the test', async function () {
```

Comment:
Changed it to `it` because the title gives a bit more info immediately when looking at the error in CI.